utils/validate-install-plan.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+const { fetchPaginatedGHResults } = require('./github-api-helpers');
+const { findMainInstallConfigFiles, readQuickstartFile } = require('./helpers');
+const path = require('path');
+
+const url = process.argv[2];
+
+const CONFIG_REGEXP = new RegExp('quickstarts/.+/config.+(yml|yaml|json)');
```

Comment:
Do we need to include `.json` files here? 

I'd expect the helper `findMainQuickstartConfigFiles` to include `json` if that was a config file type, and I'm not seeing any `config.json` files in the repo:
![image](https://user-images.githubusercontent.com/20293876/146987511-6ac54fb8-b8a6-4c1e-bbbb-8a1acab01688.png)

Reply:
my interpretation from this block of code: 
https://github.com/tyreer/newrelic-quickstarts/blob/a6ac2bd2122e45f602ebd9cd061e8047e2f02f26/utils/helpers.js#L30-L38
is that it is possible to have a `config.json`

but that might just be because of the schema jsons that were used previously to validate quickstarts??

Reply:
there is currently nothing verifying whether a config file is indeed a `json` or `yaml`, so it appears that theoretically it could be both, but we could begin to enforce that

Reply:
I read that `readQuickstartFile` function as allowing any quickstart file to be read/parsed, not limited to config files (e.g. `quickstarts/ruby/excon/dashboards/excon.json`)

Whereas the config files seem to be just YAML. Admittedly also doing a bit of interpretation of `findMainQuickstartConfigFiles` here 
https://github.com/tyreer/newrelic-quickstarts/blob/main/utils/helpers.js#L120-L127